src/modules/features/selectors.ts

```diff
@@ -4,8 +4,16 @@ import { RootState } from '../reducer'
 import { FeatureName } from './types'
 
 export const getIsCreationsTabEnabled = (state: RootState) => {
+  return false
```

Comment:
A wild return has appeared!
```suggestion
  return false
```

Reply:
my baddd i was testing combinations and missed it 😬